frontend/src/views/Apps/Import/ImportApp.js

```diff
@@ -297,8 +297,9 @@ class Import extends Component {
         <Row className="mt-2 app-networks">
           <Col className="title-page">
             <h3>Networks</h3>
-            <BootstrapTable
-              classes="table app-table app-table-empty table-striped"
+            <AppTable
+              scroll
+              toggle={[].length > 0}
```

Comment:
Missing **app-table-empty** css class. Why?

Reply:
In this case, this table is always empty, so, by default we don't need to expand/collapse.